src/main/scala/pa/model.scala

```diff
@@ -111,7 +117,7 @@ case class LeagueTeam(
 
 trait FootballMatch {
   def id: String
-  def date: DateTime
+  def date: LocalDateTime
```

Comment:
Shouldn't all these be `ZonedDateTime` ?

Reply:
https://github.com/guardian/pa-football-client/blob/master/README.md#timezones Does this not suggest LocalDateTime?

Reply:
Not to my understanding, as if we use LocalDateTime then we lose the time zone. So clients will have to force the time zone to Europe/London every time there's an interaction with a date coming from this library.

>Your application will need to be aware of this because Europe/London changes between UTC and UTC+1 depending on the time of year (BST).

I think this was in the context of migrating from the client version 4 to version 5 when zones where introduced into the client. That could be removed and I believe we need to add the zones back otherwise as it stands the client will introduce regressions

Reply:
OK. I can update that